files-community/Files
# Code Quality: Refactor Status Center code

## Feature: Display transfer speed in status center

### Motivation & Context

- Removed IStatusCenterAction interface used to access viewmodel because we now have Dependency injection.
- Removed PostedStatusBanner class and merged it to StatusBanner class which was renamed to StatusCenterItem
- Renamed some classes to use same words
- Updated UI

**Validation**
How did you test these changes?
  - Open Files
  - Copy/move/delete files
  - Open StatusCenter
  - You see what has been changed
   
### PR TO-DO

- [x] Remove InfoBar
- [x] Add color visual states

### Concepts

![image](https://github.com/files-community/Files/assets/62196528/200e9a96-b43d-4158-853c-d6b8d6491534)

### Screenshots

![image](https://cdn.discordapp.com/attachments/1147801131154685963/1147801131846729769/c.png)
